### PR TITLE
Add endpoints related to aliases

### PR DESCRIPTION
**Pull request by tgfassina**
- add proxy endpoint so that we can discover wallet address from an alias
- add js endpoints for alias/buy and alias proxy
- tests for go and js

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: server/routes/proxy-query-alias.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package routes
```
Comment (tgfassina) [inline server/routes/proxy-query-alias.go:1]: hello @6missedcalls 
could you review this when you have the chance?
thanks

File: server-in-memory/js.test.js [hunk 2]
```diff
@@ -119,6 +154,43 @@ it("fetches a list of schemas", async () => {
 	)
 })
 
+it("creates a bucket", async () => {
```
Comment (tgfassina) [inline server-in-memory/js.test.js:157]: no changes on these next 2 tests. just moved them to a place where it makes more sense

**Comment by joshLong145** [comment]
@ntindle this pull request will introduce new bypasses in the api from the motor.

**Comment by ntindle** [comment]
Rename the files to reflect the intended behavior of the endpoint not that it’s a proxy for something

**Review by Stefuu — APPROVED** [review]

**Pull request merged**